### DISCUSSION TITLE
how to ignore prototype nested in the list

### DISCUSSION THREAD

**Discussion by yanpeipan** [mapstruct/mapstruct#3185] [Q&A]
```
@Mapping(target = "list.id", ignore = true)

@Data
class S {
    List<L> list;
}

@Data
class L {
    private String id;
}
```

**Comment by filiphr** [comment]
Currently MapStruct does not offer a way to customize the mappings for nested target collection properties.

This means that you would have to define a custom mapping method from the source type to `L` and then do `@Mapping(target = "id", ignore = true)`
